### ISSUE TITLE
Front end page load size increased

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#3065]
Reported here https://github.com/CalderaWP/Caldera-Forms/pull/3052#issuecomment-470824631

**Comment by vaka2vaka** [comment]
One page
Caldera 1.7.6 page size 19,14 KB
Caldera 1.8.2 Page Size 28.84 KB

**Comment by Shelob9** [comment]
We need to audit if we're loading uneeded coded in new webpack bundle or enqueuing too many WordPress dependencies or both. Also, can we start removing cf1 files (ajax core, conditionals, etc.) in 1.8.3?

**Comment by Shelob9** [comment]
Yah, we're bundling dependencies that are redundant to WordPress. This can be fixed in webpack config.

**Comment by Shelob9** [comment]
Realted: https://stackoverflow.com/questions/33154285/exclude-react-from-webpack-bundle/41378514#41378514

As of ce58879 ~100k reduced

**Comment by vaka2vaka** [comment]
This is a template problem.
Problem in GeneratePress
In OceanWP the problem disappears.
Thank

**Comment by ictbeheer** [comment]
I don't think it is purely a theme problem. On a testsite with WP 5.1 and only Caldera Forms (and Classic Editor) activated the number of enqueued JS-files increases dramatically between CF 1.8.1 and 1.8.2:
1.8.1:  9 JS-files /  550 KB
1.8.2:  47 JS-files / 3.4 MB

**Comment by vaka2vaka** [comment]
**ictbeheer**  Yes you are right.   CF1.8.2 creates problems.

**Comment by salihkulangara** [comment]
This issue seems not related to any specific theme/template, I am using different themes for my clients, after the latest update of Caldera forms facing this issue, load a lot of js from wp-includes mostly related to react(seems related to editor). This heavily impact on the Performance, so for now I had to roll back to the version 1.7.7

**Comment by Shelob9** [comment]
New PR #3113

**Comment by Shelob9** [comment]
In #3126 we found that the blocks client was loading when it was not needed. That has a lot of extra dependencies, so that fix in 1.8.4 should help.
